pkg/runner/cypress.go

```diff
@@ -83,20 +83,24 @@ func (r *CypressRunner) Run(execution testkube.Execution) (result testkube.Execu
 		return result, fmt.Errorf("passing cypress test as single file not implemented yet")
 	}
 
-	path := filepath.Join(r.Params.Datadir, "repo", execution.Content.Repository.Path)
-	if _, err := os.Stat(filepath.Join(path, "package.json")); err == nil {
+	runPath := filepath.Join(r.Params.Datadir, "repo", execution.Content.Repository.Path)
+	if execution.Content.Repository.WorkingDir != "" {
+		runPath = filepath.Join(r.Params.Datadir, "repo", execution.Content.Repository.WorkingDir)
+	}
+
+	if _, err := os.Stat(filepath.Join(runPath, "package.json")); err == nil {
 		// be gentle to different cypress versions, run from local npm deps
-		out, err := executor.Run(path, "npm", nil, "install")
+		out, err := executor.Run(runPath, "npm", nil, "install")
```

Comment:
Just thinking if it have sense to put this inside Run?

Reply:
But then we should pass quite a lot info to generate the path :/

Reply:
:)